.github/workflows/cypress.yml

```diff
@@ -1,20 +1,36 @@
-name: "CYPRESS e2e"
+name: "ZITADEL e2e Tests"
 
 on:
-#  schedule:
-#    - cron: '*/15 * * * *'
+  schedule:
+    - cron: '*/15 * * * *'
   workflow_dispatch:
 
-
-
-env:
-  WORKING_DIR: ./
-
 jobs:
   test:
     runs-on: ubuntu-18.04
     steps:
-      - name: Checkout repository
+      - name: Checkout Repository
         uses: actions/checkout@v2
-      - name: Build the docker-compose stack
-        run: COMPOSE_DOCKER_CLI_BUILD=1 docker compose --verbose --file ${WORKING_DIR}/e2e/docker-compose-workdir.yaml --file ${WORKING_DIR}/docs/docs/guides/installation/run/docker-compose.yaml --file ${WORKING_DIR}/e2e/docker-compose-overwrite.yaml --env-file ${WORKING_DIR}/e2e/compose.env
+      - name: Set TAG env manual trigger
+        if: github.event_name == 'workflow_dispatch'
+        run: echo "RELEASE_VERSION=${{ github.event.inputs.releaseversion }}" >> $GITHUB_ENV
+      - name: Set TAG env on release
+        if: github.event_name == 'release'
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Install NPM Modules
+        working-directory: ./console
+        run: npm ci
+      - name: Test
```

Comment:
why do we need to install console?

dont we want to test the release binary?

Reply:
i`d put the testing framework outside of the console folder

Reply:
That does not answer my question.

Why do we need to install npm dependencies? are those from cypress?

The location is another topic to discuss.

Reply:
We need to install cypress dev dependencies. unfortunately, npm ci doesn't allow installing specific dependencies only https://docs.npmjs.com/cli/v8/commands/npm-ci.

But with npm ci we have at least deterministic versions. Maybe we can script that out to install specific dependencies listed in package-lock.json to avoid downloading too many modules

Reply:
And why dont we do this in a container?

Reply:
We do

Reply:
Then why is it needed in the actions file?

Reply:
the separation of console and cypress has been done